cam_width: 640
cam_height: 480
cam_fx: 603.27978515625
cam_fy: 601.8341674804688
cam_cx: 312.32598876953125
cam_cy: 237.23204040527344